rts/Lua/LuaConstCMD.cpp

```diff
@@ -82,7 +161,70 @@ bool LuaConstCMD::PushEntries(lua_State* L)
 	PUSH_CMD(PATROL);
 	/*** @field CMD.FIGHT 16 */
 	PUSH_CMD(FIGHT);
-	/*** @field CMD.ATTACK 20 */
+
+	/***
+	 * @field CMD.ATTACK 20
+	 *
+	 * Attack command. Gives an order to attack some target(s).
+	 *
+	 * The command has different modes of operation, depending on the number
+	 * of parameters and options used.
+	 *
+	 * ## Modes of operation
+	 *
+	 * ### Attack single target
+	 *
+	 * - `params` {unitID/featureID}: Attack a unit or feature.
+	 *
+	 * The command will end once the target is dead or not valid any more.
```

Comment:
Ideally would explain the conditions for "valid" later on, but this should be intuitive enough for a starter.